core/vibes/soul/primitives/button-link/button-link.makeswift.tsx

```diff
@@ -0,0 +1,49 @@
+import { Link, Select, Style, TextInput } from '@makeswift/runtime/controls';
+
+import { runtime } from '~/lib/makeswift/runtime';
+
+import { ButtonLink, Props } from '.';
+
+interface MSButtonLinkProps extends Omit<Props, 'href'> {
+  link: { href?: string; target?: string };
+  text: string;
+}
+
+runtime.registerComponent(
+  function MSButtonLink({ link, text, ...props }: MSButtonLinkProps) {
+    return (
+      <ButtonLink href={link.href ?? ''} target={link.target ?? ''} {...props}>
+        {text}
+      </ButtonLink>
+    );
+  },
+  {
+    type: 'primitive-button',
+    label: 'Primitives / Button',
```

Comment:
Should we call this `Primitives / Button Link` to be more specific?

Reply:
We've always registered a "link" version of the button, which is the only button registered in apps. Just confirmed with Andrew, we'll just call it Button for this